server/viame_tasks/utils.py

```diff
@@ -67,3 +71,13 @@ def organize_folder_for_training(
     shutil.move(str(downloaded_groundtruth), groundtruth)
 
     return groundtruth
+
+
+def get_source_video(input_path: Path, folderId: str, girder_client):
+    folder_contents = girder_client.listItem(folderId)
+    for item in folder_contents:
+        file_name = os.path.join(input_path, item.get("name"))
+        extension = os.path.splitext(file_name)[1].lower()
+        if item.get("meta", {}).get("codec") is None and extension in validVideoFormats:
+            return file_name
+    return None
```

Comment:
I don't like the strategy of "choose the thing that isn't tagged but has the right file extension".

It's fragile.  I think the transcoded file should be the thing we actually transmit, not the original source. 

If a user uploads a `txt` file named `.mp4`, this will explode.  However, if they used the tagged transcoded video, it's basically bulletproof because transcoding will only succeed on a well-formed video.

Transcoding is basically a litmus test  for valid video before training runs, so I think we should use the "safer" version.

We can slightly relax our compression on transcoding if that would help.  I really don't think this is a good idea.